sysutils/testdisk/Portfile

```diff
@@ -30,16 +29,13 @@ depends_lib     port:libcomerr \
 # e2fsprogs is not universal
 universal_variant no
 
-# libewf 2017 API changed slightly. Testdisk has not yet been updated for this
-patchfiles      patch-testdisk-new-libewf.diff
-
 use_bzip2       yes
 
 homepage        http://www.cgsecurity.org/wiki/TestDisk
 master_sites    http://www.cgsecurity.org/
 
-checksums           rmd160  56ff652a9ddf1c2b5cf96493e9aed8e7f7942d1f \
-                    sha256  00bb3b6b22e6aba88580eeb887037aef026968c21a87b5f906c6652cbee3442d
+checksums           rmd160  481f3ec24e3d2ed3b15a8efb038abf88db9bc55d \
+                    sha256  1413c47569e48c5b22653b943d48136cb228abcbd6f03da109c4df63382190fe
```

Comment:
Please add the size field to the list of checksums

Reply:
@cjones051073 This should be working now.